songs/jeff_buckley/hallelujah.sg

```diff
@@ -16,8 +15,8 @@
     That \[C]David played and it \[Am]pleased the Lord
     But \[F]you don't really \[G]care for music, \[C]do you?\[G]
     And it \[C]goes like this: the \[F]fourth, the \[G]fifth
-    The \[Am]minor fall and the \[F]major \[G]lift
-    The \[G]baffled king com\[Em]posing Halle\[Am]lujah
+    The \[Am]minor fall and the \[F]major lift
```

Comment:
Bonjour! 

Je pense que la version correcte est: 

'''
The \[Am]minor fall and the \[F]major \[G]lift
The \[Em7]baffled king composing Halle\[Am]lujah
'''